### PR TITLE
Rename get_K to get_B

### PR DESCRIPTION
**Pull request by duncanhobbs**
This PR addresses issue \#469 by renaming the function `get_K` to `get_B` in `aggregates.py`, `test_aggregates.py`, `TPI.py` and `SS.py`.

Apologies that the commit history is slightly messy. I had to resolve some merge conflicts and I am not sure I did that the most efficient way.

Let me know if I need to adjust anything. I tried to be conservative with what I changed.

@jdebacker

### CODE REVIEW

File: ogusa/SS.py [hunk 1]
```diff
@@ -235,23 +235,65 @@ def SS_solver(bmat, nmat, r, BQ, T_H, factor, Y, p, client,
               fsolve_flag=False):
     '''
     Solves for the steady state distribution of capital, labor, as well
-    as w, r, T_H and the scaling factor, using functional iteration.
```
Comment (jdebacker) [inline ogusa/SS.py]: @duncanhobbs Here you are deleting the wrong thing when there was a merge conflict.  You want to keep the HEAD in this conflict case (here, that means the docstrings that follow the Google docstring style).

File: ogusa/aggregates.py [hunk 2]
```diff
@@ -90,12 +90,12 @@ def get_I(b_splus1, K_p1, K, p, method):
...
                  (1 + np.squeeze(np.hstack((p.g_n[1:p.T], p.g_n_ss)))))
         aggI = ((1 + np.squeeze(np.hstack((p.g_n[1:p.T], p.g_n_ss)))) *
-                np.exp(p.g_y) * (K_p1 - part2) - (1.0 - p.delta) * K)
+                np.exp(p.g_y) * (B_p1 - part2) - (1.0 - p.delta) * B)
 
     return aggI
 
 
-def get_K(b, p, method, preTP):
+def get_B(b, p, method, preTP):
```
Comment (jdebacker) [inline ogusa/aggregates.py]: There are a couple instances of `K` needing to be changed to `B` in this docstring.

File: ogusa/aggregates.py [hunk 3]
```diff
@@ -43,7 +43,7 @@ def get_L(n, p, method):
     return L
 
 
-def get_I(b_splus1, K_p1, K, p, method):
+def get_I(b_splus1, B_p1, B, p, method):
```
Comment (jdebacker) [inline ogusa/aggregates.py]: The arguments in this function should still be `K_p1` and `K`.

File: ogusa/aggregates.py [hunk 4]
```diff
@@ -235,7 +235,7 @@ def get_C(c, p, method):
     return aggC
 
 
-def revenue(r, w, b, n, bq, c, Y, L, K, factor, theta, etr_params,
+def revenue(r, w, b, n, bq, c, Y, L, B, factor, theta, etr_params,
```
Comment (jdebacker) [inline ogusa/aggregates.py]: Thus function didn't need to change.

File: ogusa/aggregates.py [hunk 5]
```diff
@@ -315,7 +315,7 @@ def revenue(r, w, b, n, bq, c, Y, L, K, factor, theta, etr_params,
     return REVENUE, T_I, T_P, T_BQ, T_W, T_C, business_revenue
 
 
-def get_r_hh(r, r_gov, K, D):
```
Comment (jdebacker) [inline ogusa/aggregates.py]: This function should not change.  `K` is the appropriate argument.

File: ogusa/tests/test_aggregates.py [hunk 6]
```diff
@@ -44,15 +44,15 @@ def test_get_L(n, p, method, expected):
 # update parameters instance with new values for test
 p.update_specifications(new_param_values)
 b_splus1 = 10 * np.random.rand(p.T * p.S * p.J).reshape(p.T, p.S, p.J)
-K_p1 = 0.9 + np.random.rand(p.T)
```
Comment (jdebacker) [inline ogusa/tests/test_aggregates.py]: This doesn't need to change.

File: ogusa/tests/test_aggregates.py [hunk 7]
```diff
@@ -63,18 +63,18 @@ def test_get_L(n, p, method, expected):
...
-             (b_splus1, K_p1, K, p, 'TPI', aggI_TPI)]
+            * np.exp(p.g_y) * (B_p1 - part2) - (1.0 - p.delta) * B)
+test_data = [(b_splus1[-1, :, :], B_p1[-1], B[-1], p, 'SS', aggI_SS),
+             (b_splus1, B_p1, B, p, 'TPI', aggI_TPI)]
 
 
-@pytest.mark.parametrize('b_splus1,K_p1,K,p,method,expected', test_data,
+@pytest.mark.parametrize('b_splus1,B_p1,B,p,method,expected', test_data,
                          ids=['SS', 'TPI'])
-def test_get_I(b_splus1, K_p1, K, p, method, expected):
```
Comment (jdebacker) [inline ogusa/tests/test_aggregates.py]: This doesn't need to change.

File: ogusa/tests/test_aggregates.py [hunk 8]
```diff
@@ -92,27 +92,27 @@ def test_get_I(b_splus1, K_p1, K, p, method, expected):
...
 test_data = [(b[-1, :, :], p, 'SS', expected1),
              (b, p, 'TPI', expected2)]
 
 
 @pytest.mark.parametrize('b,p,method,expected', test_data,
                          ids=['SS', 'TPI'])
-def test_get_K(b, p, method, expected):
+def test_get_B(b, p, method, expected):
     """
     Test aggregate capital function.
```
Comment (jdebacker) [inline ogusa/tests/test_aggregates.py]: In docstring for this test. say "Test aggregate savings function."

File: ogusa/tests/test_aggregates.py [hunk 9]
```diff
@@ -218,7 +218,7 @@ def test_get_C(c, p, method, expected):
 bq = BQ / p.lambdas.reshape(1, 1, p.J)
 Y = 0.561 + (0.602 - 0.561) * random_state.rand(p.T).reshape(p.T)
 L = 0.416 + (0.423 - 0.416) * random_state.rand(p.T).reshape(p.T)
-K = 0.957 + (1.163 - 0.957) * random_state.rand(p.T).reshape(p.T)
```
Comment (jdebacker) [inline ogusa/tests/test_aggregates.py]: Nothing needs to change with the `test_get_revenue`.

File: ogusa/tests/test_aggregates.py [hunk 10]
```diff
@@ -300,13 +300,13 @@ def test_revenue(r, w, b, n, bq, c, Y, L, K, factor, theta, etr_params, p,
              (0.04, 0.02, 2.0, 0.0, 0.04)]
 
 
-@pytest.mark.parametrize('r,r_gov,K,D,expected', test_data,
```
Comment (jdebacker) [inline ogusa/tests/test_aggregates.py]: Nothing needs to change with this test.

**Comment by jdebacker** [comment]
@duncanhobbs thanks again for tackling this.  I see one instance where you chose the wrong branch in the merge conflict and a few instances where you changed functions (and tests of those functions) that did not need to change.  These include `aggregates.revenue()`, `aggregates.get_I()`, and `aggregates.get_hh_r()`.

Please see specific inline comments for more details.  Thanks!

**Comment by jdebacker** [comment]
@duncanhobbs Please let me know if my comments here are helpful or if you need more info before proceeding with this PR. Thanks!

**Comment by duncanhobbs** [comment]
@jdebacker 

I think this is clear. I am just making sure that I change everything back correctly so that the tests do not fail

Is it easier to try revert the existing commits or to open a new PR given the other changes to the master branch?

**Comment by jdebacker** [comment]
I think you can just revert some of the changes in this PR more easily.

Also, before you merge, let's make sure you can run the full suite of tests locally (due to runtime constraints, only a subset of the tests run on Travis CI).

Thanks!

**Comment by rickecon** [comment]
@jdebacker @duncanhobbs . Is this PR ready to go? It looks like Jason made a PR to Duncan's PR branch, and the changes look good to me. I will merge this as soon as I get the green light. Then I will merge PR #479 (Transfers).

**Comment by jdebacker** [comment]
Yes - ready to merge!  Thanks @rickecon.

After this, I'll sync the `transfers` brach in PR #479 then it should be ready to merge also.

**Pull request merged**